[4.0] bye bye xref

brianteeman opened the pull request:
WIP

Pull Request for Issue #25272 

Documents it, Explain it or just kill it

No point having code no one knows what to do with it.

We have enough fields to confuse people without two extra fields

brianteeman committed:
[4.0] bye bye xref

WIP

Quy commented on components/com_content/Model/ArticleModel.php:
```diff
@@ -104,7 +104,7 @@ public function getItem($pk = null)
 							'a.modified_by, a.checked_out, a.checked_out_time, a.publish_up, a.publish_down, ' .
 							'a.images, a.urls, a.attribs, a.version, a.ordering, ' .
-							'a.metakey, a.metadesc, a.access, a.hits, a.metadata, a.featured, a.language, a.xreference'
+							'a.metakey, a.metadesc, a.access, a.hits, a.metadata, a.featured, a.language
```
```suggestion
							'a.metakey, a.metadesc, a.access, a.hits, a.metadata, a.featured, a.language'
```

brianteeman committed:
Update components/com_content/Model/ArticleModel.php

Co-Authored-By: Quy <quy@fluxbb.org>

Quy commented on administrator/components/com_content/forms/article.xml:
```diff
@@ -654,14 +654,6 @@
-		name="xreference"
-		type="text"
-		label="JFIELD_KEY_REFERENCE_LABEL"
-		description="JFIELD_KEY_REFERENCE_DESC"
```
Remove language strings.

brianteeman committed:
string

Quy commented on administrator/components/com_contact/sql/install.mysql.utf8.sql:
```diff
@@ -54,5 +53,4 @@ CREATE TABLE IF NOT EXISTS `#__contact_details` (
   KEY `idx_createdby` (`created_by`),
   KEY `idx_featured_catid` (`featured`,`catid`),
   KEY `idx_language` (`language`),
```
```suggestion
  KEY `idx_language` (`language`)
```

Quy commented on administrator/components/com_newsfeeds/sql/install.mysql.utf8.sql:
```diff
@@ -40,5 +39,4 @@ CREATE TABLE IF NOT EXISTS `#__newsfeeds` (
   KEY `idx_catid` (`catid`),
   KEY `idx_createdby` (`created_by`),
   KEY `idx_language` (`language`),
```
```suggestion
  KEY `idx_language` (`language`)
```

brianteeman committed:
Update administrator/components/com_contact/sql/install.mysql.utf8.sql

Co-Authored-By: Quy <quy@fluxbb.org>

brianteeman committed:
Update administrator/components/com_newsfeeds/sql/install.mysql.utf8.sql

Co-Authored-By: Quy <quy@fluxbb.org>

wilsonge commented:
It's unused kill it

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 1/16  (#25450)

brianteeman committed:
[4.0] behavior.hastooltip (#25501)

This PR removes the remaining instances of `HTMLHelper::_('behavior.tooltip');`

It doesn't do anything at all as can be seen
https://github.com/joomla/joomla-cms/blob/42067d565666ac51d01b8b96d422e294cc494da4/libraries/cms/html/behavior.php#L112

It is just a copy/paste error

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 2/16? (#25451)

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 3/16 (#25453)

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 4/16?  (#25455)

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 5/16 (#25456)

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 6/16  (#25457)

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 7/16?  (#25458)

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 8/16? (#25459)

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 9/16? (#25462)

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 10/16 (#25464)

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 11/16? (#25465)

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 12/16? (#25466)

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 13/16? (#25468)

puneet0191 committed:
Updating Localhost testing setup instructions, adding selenium standalone node module (#25483)

brianteeman committed:
[4.0] Unused file (#25513)

afaict this file stopped being used in 2013 when the reference to loadTemplate('options') was removed with this commit 7a9822cd1ab9445a5db7f5a12c577e0b4809f1cf

assuming i am correct then it would be a good idea to stop maintaining a ghost file

Note: There may be more

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 14/16? (#25469)

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 15/16? (#25470)

Hackwar committed:
[4.0] Codestyle fixes for new CMS codestyle Batch 16/16 (#25471)

Hackwar committed:
[4.0] Add an Integration test suite stub (#25118)

Hackwar commented:
Bye bye

wilsonge committed:
Fix codestyle

wilsonge committed:
Revert back CS Change as it fails anyway and was 'correct' before

wilsonge committed:
Fix backslash to forward slash

wilsonge committed:
Update coding standards to fix closures

wilsonge committed:
Fix input filter codestyle

HLeithner committed:
Update .done.yml signature (#25528)

wilsonge committed:
Fix line lengths

wilsonge committed:
Change doc block to satisfy code standards

Hackwar committed:
Fixing case of file (#25530)

wilsonge committed:
Fix application being injected into the router

wilsonge committed:
[4.0] Disallow failures on new codestyle (#25529)

Commit:
Remove deprecated *_dev_level attributes from update server manifests (#25495)

infograf768 committed:
[4.0] Correcting massmail js (#25517)

Commit:
Namespace JDate uses (#25512)

Commit:
[4.0] Fix Actionlog model loading (#25511)

Commit:
Always load language files (#25535)

810 committed:
[4.0] Codeception codestyle (#25523)

brianteeman commented:
conflicts resolved

wilsonge commented:
Something went wrong with conflict resolution :/

brianteeman commented:
grrh

brianteeman commented:
See #25549
